.github/workflows/deploy.yml

```
@@ -9,9 +9,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+      - uses: pnpm/action-setup@v2
+        with:
+          version: 8.12.0
       - name: Publish
         uses: cloudflare/wrangler-action@v3
         with:
           apiToken: ${{ secrets.CF_API_TOKEN }}
         env:
-          CF_ACCOUNT_ID: ${{ secrets.CF_ACCOUNT_ID }}
+          CLOUDFLARE_ACCOUNT_ID: ${{ secrets.CF_ACCOUNT_ID }}
```

I think this change is not necessary. Cloudflare [uses `CF_ACCOUNT_ID` everywhere](https://developers.cloudflare.com/search/#q=CF_ACCOUNT_ID).

```suggestion
          CF_ACCOUNT_ID: ${{ secrets.CF_ACCOUNT_ID }}
```